jwql/utils/constants.py

```diff
@@ -963,3 +964,11 @@ class QueryConfigKeys:
     "nirspec": "https://jwst-docs.stsci.edu/jwst-near-infrared-spectrograph",
     "nircam": "https://jwst-docs.stsci.edu/jwst-near-infrared-camera",
 }
+
```

Comment:
Generally we've been trying to keep the constants in alphabetical order. It looks like we're not at 100% for this, but it could help to move these definitions up in the file.

Reply:
Fixed.